net/BUILD.gn

```diff
@@ -3302,15 +3303,17 @@ test("net_unittests") {
     ]
   }
 
+  if (use_cobalt_customizations) {
+    sources -= [
+      # NetworkInterfaces is not implemented by Starboard
```

Comment:
Instead of saying it's not implemented let's say it's stubbed out, it was confusing for me to read this while seeing the corresponding source added above